config/samples/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ resources:
 - dataplane_v1beta1_openstackdataplanenode_edpm_compute_0.yaml
 - dataplane_v1beta1_openstackdataplanenode_edpm_compute_1.yaml
 - dataplane_v1beta1_openstackdataplanenode_edpm_compute_0_full.yaml
-- dataplane_v1beta1_openstackdataplane.yaml
+- dataplane_v1beta1_openstackdataplane_sample.yaml
```

Comment:
dataplane_v1beta1_openstackdataplane does have jinja vars: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplane.yaml#L114-L119
I made a copy of it, but removing the vars and the CI got fixed

Reply:
Nice find. Instead of removing the vars though, can we just expand them to what their default values would be? Then include that version in the CSV.

Reply:
done!